src/views/activity/components/transaction-card/transaction-card.view.tsx

```diff
@@ -0,0 +1,114 @@
+import useTransactionCardStyles from "src/views/activity/components/transaction-card/transaction-card.styles";
+import { ReactComponent as TransferL1Icon } from "src/assets/icons/transfer-l1.svg";
+import { ReactComponent as TransferL2Icon } from "src/assets/icons/transfer-l2.svg";
+import { ReactComponent as ReloadIcon } from "src/assets/icons/reload-card.svg";
+import { ReactComponent as EthToken } from "src/assets/tokens/eth.svg";
+import { ReactComponent as DaiToken } from "src/assets/tokens/dai.svg";
+import FiatAmount from "src/views/shared/fiatAmount/fiat-amount";
+import Typography from "src/views/shared/typography/typography.view";
+import Card from "src/views/shared/card/card.view";
+
+export interface TransactionCardProps {
+  type: "l1" | "l2";
+  id: number;
+  timestamp: number;
+  token: "eth" | "dai";
+  status: "hold" | "initiate" | "process" | "complete" | "error";
+  amount: number;
+}
+
+const TrasnsactionCard = ({ type, timestamp, token, amount, status }: TransactionCardProps) => {
+  const classes = useTransactionCardStyles();
+  const IconsLayer = {
+    l1: TransferL1Icon,
+    l2: TransferL2Icon,
+  };
+  const Icon = status !== "complete" && status !== "error" ? ReloadIcon : IconsLayer[type];
```

Comment:
We need to create a shared component called `Spinner` to represent the spinning effect, as it needs to spin. I wouldn't add the animation by the moment as Chami asked for a very specific animation, but I would create the component and move this condition to the JSX.

Reply:
But one thing it's the component and other the logic that I think that should be here because the shared spinner could have differents status that these ones. 
While we don't have the animation I'm not see necessary separate it